### ISSUE TITLE
Can we write `ImageSeries` in external_mode if the timestamps are well specified

### ISSUE THREAD

**Issue by h-mayorquin** [catalystneuro/neuroconv#472]
Right now when external mode is chosen we have the starting frames as a requirement:

https://github.com/catalystneuro/neuroconv/blob/2da9f6bef30b46a52b3303d41eb6f80cf8cccffa/src/neuroconv/datainterfaces/behavior/video/videodatainterface.py#L327-L329

That is, for ImageSeries of multiple videos in external mode, we add the requirements of starting_frames. Is this necessary? I can't see any of this specified on pynwb:

https://pynwb.readthedocs.io/en/stable/pynwb.image.html

What is the logic here? 
My feeling is that if the timestamps are well specfiied then we should be able to write the ImageSeries.

**Comment by CodyCBakerPhD** [comment]
The `timestamps` don't tell you the number of frames in each external video file

As far as PyNWB, no I don't think it's a strict requirement, or even technicall ya  best practice just yet

The closest mention we get is in the code https://github.com/NeurodataWithoutBorders/pynwb/blob/a6fa9170f2ad6942d74eaf44c22cedf0b80f0fcd/src/pynwb/image.py#L82-L90

**Comment by CodyCBakerPhD** [comment]
It's not just about the timing of the frames, it's valuable context about how the shape of the total ImageSeries is distributed across the external files

Example: one very short video and one very long one - nearly contiguous but slightly irregular timestamps - timestamps alone do not tell you exactly which frames of the timestamp vector correspond to which video file

**Comment by h-mayorquin** [comment]
OK, thanks, I undestand that is very important information.

The logic as I understand it now is that: an external file with wihout information would be very incomplete, so we don't let you, even if you could.

**Comment by h-mayorquin** [comment]
Right, I guess we are moving towards there in https://github.com/catalystneuro/neuroconv/pull/464

**Comment by h-mayorquin** [comment]
OK, I am closing this issue as now we have separated this in the video interface

https://github.com/catalystneuro/neuroconv/pull/1251

Maybe we will re-open a similar one in the future about frames in metadata.
